src/components/Match/matchColumns.jsx

```diff
@@ -806,6 +806,14 @@ export default (strings) => {
             </span>}
           </div>),
     },
+    {
+      displayName: strings.cs_over_time,
+      tooltip: strings.tooltip_cs_over_time,
+      field: 'cs_t',
+      sparkline: true,
+      strings: strings,
```

Comment:
It feels weird to pass the whole strings object here, but in order to localize the text of the tooltip in the graph that's a table element, it seems necessary. =/